FlatBuffersSwift/FlatBuffersBuilder.swift

```diff
@@ -172,51 +237,78 @@ public final class FBBuilder {
         objectStart = Int32(cursor)
     }
     
+    /**
+     Append an offset into the buffer for the currently open object
+     
+     - parameters:
+         - propertyIndex: The index of the property to update
+         - offset: The offsetnumber of properties we will update
+
+     - Returns: The current cursor position (Note: What is the use case of the return value?)
+     */
     @discardableResult
-    public func addPropertyOffsetToOpenObject(propertyIndex : Int, offset : Offset) throws -> Int{
+    public func append(propertyIndex : Int, offset : Offset) throws -> Int{
```

Comment:
Do you want to reverse the arguments?

Looking at usage, current proposal:

`        let sOffset = try! fbb.append(value: "max")
        try! fbb.startObject(numOfProperties: 3)
        try! fbb.append(propertyIndex: 0, value: true, defaultValue: false)
        try! fbb.append(propertyIndex: 1, offset: sOffset)
        let oOffset = try! fbb.endObject()
        try! fbb.finish(offset: oOffset, fileIdentifier: nil)
`

would become:

`        let sOffset = try! fbb.append(value: "max")
        try! fbb.startObject(numOfProperties: 3)
        try! fbb.insert(value: true, defaultValue: false, toStartedObjectAt: 0)
        try! fbb.insert(offset: sOffset, toStartedObjectAt: 1)
        let oOffset = try! fbb.endObject()
        try! fbb.finish(offset: oOffset, fileIdentifier: nil)
`

Perhaps:
`        let sOffset = try! fbb.append(value: "max")
        try! fbb.startObject(numOfProperties: 3)
        try! fbb.update(value: true, defaultValue: false, toStartedObjectAt: 0)
        try! fbb.update(offset: sOffset, toStartedObjectAt: 1)
        let oOffset = try! fbb.endObject()
        try! fbb.finish(offset: oOffset, fileIdentifier: nil)
`

?